### DISCUSSION TITLE
How do I supposed to run it on my infrastructure?

### DISCUSSION THREAD

**Discussion by leandro-toledo-uux** [hoppscotch/hoppscotch#2061] [Q&A] [answered]
Since the [Release section ](https://github.com/hoppscotch/hoppscotch#releasing) says we would take into account the /dist folder and the [Dockerfile](https://github.com/hoppscotch/hoppscotch/blob/main/Dockerfile) execs the CMD ["pnpm", "run", "start"]. It was not clear to me, if I want to host it on my infra, do I need http serverside listening some port (nginx proxy like), OR can I only send static files (html and js) and it is rendered in the client.... (if we are able to do the 2º option, what happens with firebase variables (.env) are they used only in compile time?)

Thanks in advance

**Comment by liyasthomas** [comment]
Hoppscotch is a 100% client side application and it can be hosted on any server that support hosting HTML / JAM stack websites. You can read more about hosting here: https://github.com/hoppscotch/hoppscotch/wiki/Hosting

**Comment by liyasthomas** [comment, marked as answer]
Refer [Hoppscotch documentation](https://docs.hoppscotch.io) for instructions to set up Hoppscotch locally.
